src/iuse.cpp

```diff
@@ -5556,7 +5562,7 @@ int iuse::portal(player *p, item *it, bool, const tripoint& )
     if (it->charges < it->type->charges_to_use()) {
         return 0;
     }
-    g->m.add_trap(p->posx() + rng(-2, 2), p->posy() + rng(-2, 2), tr_portal);
+    g->m.add_trap({p->posx() + rng(-2, 2), p->posy() + rng(-2, 2), p->posz()}, tr_portal);
```

Comment:
That doesn't quite explain why the 64-bit Windows builds didn't complain, but I do approve of using `long` for `point` and `tripoint`.